charts/cityos/templates/deployment.yaml

```diff
@@ -47,19 +36,23 @@ spec:
             - name: SPRING_PROFILES_ACTIVE
               value: "prod"
             - name: SPRING_DATASOURCE_URL
-              value: jdbc:mariadb://{{ .Release.Name }}-mariadb:3306/{{ .Values.mariadb.auth.database }}?useSSL=false&serverTimezone=UTC&useLegacyDatetimeCode=false
```

Comment:
I think it would be useful to change `postgresql` to something more descriptive like `cityosapp-postgresql`. Otherwise, it will be hard to integrate a second postgres into the chart

Reply:
ich verstehe nicht was du meinst. Der Datenbankname ist per values konfigurierbar